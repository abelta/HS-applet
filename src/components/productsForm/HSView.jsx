import React from 'react';
import PropTypes from 'prop-types';

const HSView = ({ value }) => (
  <div>
    <h3>Selected HS Code</h3>
    <p>{value}</p>
  </div>
);

HSView.defaultProps = {
  value: '',
};

HSView.propTypes = {
  value: PropTypes.string,
};

export default HSView;
