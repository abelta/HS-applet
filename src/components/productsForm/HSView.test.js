import React from 'react';
import renderer from 'react-test-renderer';
import HSView from './HSView';

describe('HSView', () => {
  const tree = renderer.create(<HSView value={[1, 2, 3]} />).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
