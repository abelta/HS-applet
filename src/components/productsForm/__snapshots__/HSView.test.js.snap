// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HSView renders correctly 1`] = `
<section
  className="row"
  style={
    Object {
      "height": "100px",
    }
  }
>
  <span
    className="col s4 center-align"
    style={
      Object {
        "fontSize": 80,
        "fontWeight": "bold",
        "lineHeight": "100px",
      }
    }
  >
    1
  </span>
  <span
    className="col s4 center-align"
    style={
      Object {
        "fontSize": 80,
        "fontWeight": "bold",
        "lineHeight": "100px",
      }
    }
  >
    2
  </span>
  <span
    className="col s4 center-align"
    style={
      Object {
        "fontSize": 80,
        "fontWeight": "bold",
        "lineHeight": "100px",
      }
    }
  >
    3
  </span>
</section>
`;
